client/lib/plans/features-list.js

```diff
@@ -127,6 +127,18 @@ import {
 	FEATURE_WP_SUBDOMAIN,
 	FEATURE_WP_SUBDOMAIN_SIGNUP,
 	PREMIUM_DESIGN_FOR_STORES,
+	FEATURE_TIERED_20GB_BACKUP_STORAGE,
```

Comment:
I think "tiered" is redundant here and it makes more sense to just have `FEATURE_1TB_BACKUP_STORAGE`, `FEATURE_UNLIMITED_BACKUP_STORAGE`, etc. This will also future-proof this for a time when we don't have tiers, and instead just have Backup + storage products.

Reply:
Having a `_TIERED` qualifier is one of those things that I agree isn't ideal, but I'm also unsure what a good alternative might be. I guess it wouldn't hurt to re-use existing features, even if they were originally put in to serve different products. What are your thoughts?

To paint a clearer picture of what I'm struggling with, in some situations we have multiple features available with similar copy, but intended for different products. This happens notably with real-time backups, one-click restores, and unlimited storage, but others also exist. I'm not saying we need _more_ variations -- far from it -- but my intention was to strongly tie these features for our products as they exist now, making adjustments and hopefully cleaning up the features list as we iterate. Almost certainly some of these slugs are no longer in use.

**Real-time backups:**

| Slug                                         | Copy                                   |
| -------------------------------------------: | :------------------------------------- |
| `FEATURE_BACKUP_REALTIME_V2`                 | Backup (real-time, off-site)           |
| `FEATURE_OFFSITE_BACKUP_VAULTPRESS_REALTIME` | {{strong}}Real-time{{/strong}} backups |
| `FEATURE_PRODUCT_BACKUP_REALTIME_V2`         | Backup Real-time (off-site)            |

**One-click restores:**

| Slug                               | Copy                                         |
| ---------------------------------- | -------------------------------------------- |
| `FEATURE_AUTOMATED_RESTORES`       | Automated restores                           |
| `FEATURE_ONE_CLICK_RESTORE_V2`     | One-click restores                           |
| `FEATURE_SITE_BACKUPS_AND_RESTORE` | Automated site backups and one-click restore |

**Unlimited storage:**

| Slug                                     | Copy                                      |
| ---------------------------------------- | ----------------------------------------- |
| `FEATURE_BACKUP_STORAGE_SPACE_UNLIMITED` | Unlimited backup storage space            |
| `FEATURE_SECURE_STORAGE_V2`              | Unlimited backup storage                  |
| `FEATURE_UNLIMITED_STORAGE`              | {{strong}}200 GB{{/strong}} storage space |

Reply:
In that case I would replace `TIERED` with `JETPACK`, so for example `FEATURE_TIERED_UNLIMITED_BACKUP_STORAGE` would become `FEATURE_JETPACK_UNLIMITED_BACKUP_STORAGE` to differentiate it from `FEATURE_UNLIMITED_STORAGE`, but this is just a nit so I don't have a really strong opinion. Either way I think this is good to go once p1HpG7-d65-p2 has feedback.